lib/pbench/test/unit/server/conftest.py

```diff
@@ -220,14 +220,15 @@ def compare(message: str, response: Optional[Response]):
         internal = re.compile(f"{name}Internal error {uuid_re}: {message}")
         if response:
             assert response.status_code == HTTPStatus.INTERNAL_SERVER_ERROR
-            assert external.match(response.json["message"])
+            assert external.match(response.json()["message"])
```

Comment:
Dave...could you look at this and concur, please?  (I don't know how this was "working" before....)

Reply:
Nevermind...the unit tests failed...so, my linter is suffering from some type confusion.

Reply:
OK, I've reverted this change and instead amended the `import` statement so that it pulls in what I _think_ is the correct `Response` type.  😝  The linter is still happy; we'll see if the unit tests are.

(Note, I've amended this commit...in the event that you were silly enough to look at it instead of, say, going for a walk in this beautiful weather.)

Reply:
Sadly, I didn't see your response before posting mine, but it sounds like we both came to the same conclusions, although I have one additional wrinkle.

In any case, I've removed the errant reference to `capinternal` which, as you point out, isn't actually used, so hopefully that will simplify the problem space.

> The others are all using the `client` fixture, which is `flask.wrappers.Response`.

According to my IDE, the `client` methods return a `werkzug.test.TestResponse` which is a subclass of a `werkzug.wrappers.response.Response`.  I don't know how that relates to a `flask.wrappers.Response`, but I made the `capinternal` fixture match its usage...hopefully, all will be good, now.  (You indicated that there is no difference...so, all's good, I guess...or, that is, hopefully it will be, once Quay.io comes back so that we can access the CI container....)

Reply:
We seem to import `Response` in 21 places.  In five of them, we pull it from `flask`; we pull it from `flask.wrappers` in 14 of them.

One of the other two is in `test_datasets.py`, and I'm wondering if that should be changed.

The other is the one that I just changed to `werkzug`...which I'm thinking of changing to `flask` or `flask.wrappers`...and I'm wondering which I should use....

Reply:
> One of the other two is in `test_datasets.py`, and I'm wondering if that should be changed.

Nope...that one looks correct:  it's using our Server client utility, which really does use `requests.Response`.

Reply:
`test_datasets.py` uses my API client, which is `requests` based.

Werkzeug is used throughout Flask and I doubt there's any major harm in using it directly as it's unlikely Flask would/could divest ... however in principle we have cleaner encapsulation going through Flask. But I didn't remember that `flask` exposed a `Response` directly vs `flask.wrappers`, so I suppose the direct import is "better" but the inconsistency isn't the worst possible fate and it you want to add the noise to make them consistent maybe just change whichever is used less often?